README.md

```diff
@@ -68,12 +80,29 @@ This one prints short documentation for all the commands available.
 swift-express help
 ```
 
+### Swift Package Manager commands
+
+These commands work with Swift Package Manager. They are analogs for standard commands, but with SPM as package manager.
+
+```sh
+swift-express build-deps-spm
```

Comment:
It feels a bit overloaded as well. I think we could just add a `--spm` or `--use-spm` (I prefer first one) key to all the commands and thus it will make it flat and simple. On Linux it will use it by default. For convenience we could add `--carthage` key, which will be used by default on OS X (for now, we will change this behaviour in later versions).

Another option is to use something like `--build-tool=spm|carthage`. Let's discuss what's better.